### ISSUE TITLE
Create Your First ABAP Console Application

### ISSUE THREAD

**Issue by agl-phenr** [SAPDocuments/Tutorials#3010]
Hello,

I am sticking to this link in order to hands on ABAP Cloud:
https://developers.sap.com/tutorials/abap-environment-console-application.html

I was not succeeded as it´s asking for service key in JSON format.
May I know where to get this file?

Besides this, I was wondering if ABAP cloud environment is already live? If not, may I know when it is planned to? 

We just bought SCP Neo environment.

Best Regards,
Pietro
![adt - abap cloud project](https://user-images.githubusercontent.com/42214365/51996762-507fb680-24ad-11e9-83c2-ccc280f84233.png)

**Comment by mervey45** [comment]
Hi @agl-phenr, 

you can also make you of the other way around. We can't make the service key in JSON public, because you need to copy it from your SAP Cloud Platform Cockpit. But you can make use by clicking on SAP Cloud Platform Cloud Foundry Environment. Yes, SAP Cloud Platform ABAP environment is live. But you need to buy it.

![image](https://user-images.githubusercontent.com/27953324/52043258-879ea800-253f-11e9-9e2c-3c530b866c70.png)
 
And then you can select europe as region and give your credentials. 
![image](https://user-images.githubusercontent.com/27953324/52043295-9dac6880-253f-11e9-82ae-abeeb1dfac67.png)

Best regards, 
Merve

**Comment by agl-phenr** [comment]
Hi @mervey45,

Such a great pleasure talk to you :)

I have managed go through the credentials but now it´s asking Organization. I do not see my Organization but trial account only. 
![image](https://user-images.githubusercontent.com/42214365/52043929-c5e49900-2538-11e9-8973-1b2f2b672f6d.png)
So, I am not sure if I have managed it as my trial account or my company´s SCP has something missing.


Anyway I would like kindly add a question in advanced.
The company I work is a SAP partner and they have bought a SCP license.

![image](https://user-images.githubusercontent.com/42214365/52044112-368bb580-2539-11e9-9f1a-9005a0ac2af7.png)

By logging on the SCP, I get this:
![image](https://user-images.githubusercontent.com/42214365/52044249-96825c00-2539-11e9-8c91-f9db43ec0eba.png)

My question is by having the NEO environment I will be able to use ABAP cloud? or I will be required to have CF environment or even ABAP environment?

I have read the SAP help where is said I was supposed to have a third environment (ABAP environment):

https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/ab512c3fbda248ab82c1c545bde19c78.html

![image](https://user-images.githubusercontent.com/42214365/52044940-41474a00-253b-11e9-8077-e7cb15f485b6.png)

Thank you a lot for all the help :)

Have a great day.
Pietro

**Comment by mervey45** [comment]
Hi @agl-phenr ,

you're welcome. You need to have our new offering, the SAP Cloud Platform, ABAP environment, which is currently in the Cloud Foundry environment. Otherwise it won't work. We offer trial versions only in Code Jams and Events. But we're working on that topic.

Best regards, 
Merve

**Comment by agl-phenr** [comment]
Hi @mervey45,

Got it. 
Indeed I have managed to raise a CF sub account. See below:

![image](https://user-images.githubusercontent.com/42214365/52048708-f67e0000-2543-11e9-884c-0e0c2e18da3d.png)

Back to the Eclipse, I went through all the way up to instance. See below:

![image](https://user-images.githubusercontent.com/42214365/52048810-2cbb7f80-2544-11e9-8fed-d7a988f4823c.png)

On the SCP, I got no ABAP service so my guess is I was supposed to get into the abap service and then create a service instance. Am I right? 

In advance: if I was supposed to see abap service in the service marketplace then does it mean I missed this new offering? If so, as partner we might have it already? 

Best Regards,
Pietro
